packages/airnode-node/src/evm/fulfillments/api-calls.ts

```diff
@@ -64,7 +64,7 @@ async function testFulfill(
     );
   const [err, res] = await go(operation, { retries: 1, timeoutMs: DEFAULT_RETRY_TIMEOUT_MS });
   if (err) {
-    const errorLog = logger.pend('ERROR', `Error attempting API call fulfillment for Request:${request.id}`, err);
+    const errorLog = logger.pend('ERROR', `Static call fulfillment failed for Request:${request.id} with ${err}`, err);
```

Comment:
if err is an object would this throw [object] ? Should we do JSON.stringify(err) @Siegrift ?

Reply:
Mhh, I assumed it will always be an error instance which stringifies to string. It might be better to check for this explicitely and json.stringify in case it isn't.

Reply:
I believe @Siegrift  is correct and `err` has to be an `Error` instance because the `go` [function](https://github.com/api3dao/airnode/blob/7da75b33b2a9faffdd84518a84861845c46e3309/packages/airnode-utilities/src/promises/promise.ts#L30) returns a `Promise<GoResult<T>>` where `GoResult` is:
https://github.com/api3dao/airnode/blob/7da75b33b2a9faffdd84518a84861845c46e3309/packages/airnode-utilities/src/promises/promise.ts#L10